### PR TITLE
REVIEW: get_age_model()

### PR DESCRIPTION
**Pull request by LewisAJones**
Hi @bethany-j-allen, not much to contribute here from my side! I've made a few suggested changes - let me know what you think!

### CODE REVIEW

File: tests/testthat/test-get_age_model.R [hunk 1]
```diff
@@ -7,4 +7,6 @@ test_that("get_age_model works", {
                                            column_id = "test"),
                           warn_args = list(column_id = -503),
                           col_no = 15)
+  # No args provided
+  expect_error(get_age_model())
```
Comment (willgearty) [inline tests/testthat/test-get_age_model.R:11]: You can actually test this by adding an empty list to the `fail_args` argument.

Comment (LewisAJones) [reply tests/testthat/test-get_age_model.R:11]: Doh! Of course - updated!

**Review by bethany-j-allen — APPROVED** [review]
I removed the word 'largely' from the description, because it wasn't needed. But otherwise happy to approve this.

**Comment by LewisAJones** [comment]
Thanks Bethany!

**Pull request merged**